src/operators/UserOperations.ts

```
@@ -0,0 +1,48 @@
+import User, { UserInterface } from '../models/user';
+
+export const create = async (user: UserInterface) => {
+	try {
+		console.log(user.semesters + 'id: ' + user.id);
+		await User.create(user);
+	} catch (e) {
+		throw 'ERROR: User could not be created';
+	}
+};
+export const read = async (key: string) => {
+	try {
+		const user = await User.get(key);
+		if (user) {
+			return user;
+		} else {
+			throw 'ERROR: Could not read user';
+		}
+	} catch (e) {
+		console.error(e);
+	}
```

Should you also throw an error here?